conf.yaml

```diff
@@ -1436,35 +1436,50 @@ contents:
     -
         title:      Enterprise Search
         sections:
+          -
+            title:      Enterprise Search Guide
+            prefix:     en/enterprise-search
+            index:      enterprise-search-docs/index.asciidoc
+            private:    1
+            current:    *stackcurrent
+            branches:   [ 7.7 ]
+            live:       *stacklive
+            chunk:      1
+            tags:       Enterprise Search/Guide
+            subject:    Enterprise Search
+            sources:
+              -
+                repo:   enterprise-search-pubs
+                path:   enterprise-search-docs
           -
             title:      Workplace Search Guide
             prefix:     en/workplace-search
             index:      workplace-search-docs/index.asciidoc
             private:    1
-            current:    7.6
-            branches:   [ 7.6 ]
+            current:    *stackcurrent
+            branches:   [ 7.7, 7.6 ]
             live:       *stacklive
             chunk:      1
-            tags:       Workplace Search/Reference
+            tags:       Workplace Search/Guide
             subject:    Workplace Search
             sources:
               -
                 repo:   enterprise-search-pubs
                 path:   workplace-search-docs
           -
-            title:      App Search Reference
-            prefix:     en/swiftype/appsearch
-            index:      docs/appsearch/index.asciidoc
+            title:      App Search Guide
+            prefix:     en/app-search
+            index:      app-search-docs/index.asciidoc
             private:    1
-            current:    master
-            branches:   [ master ]
-            single:     1
-            tags:       App Search/Reference
-            subject:    Swiftype
+            current:    *stackcurrent
+            branches:   [ 7.7 ]
```

Comment:
@goodroot @jonasll Same deal. Let me know if we should add `7.8` here.

Reply:
Thanks @jrodewig !

@chriscressman for visibility. He is back in action. (Enterprise Search doc contact)

Reply:
Same, please add 7.8.

Reply:
Added with 7e38646.